src/databricksbundle/DatabricksBundle.py

```diff
@@ -38,6 +40,30 @@ def __init__(self, databricks_config: str):
     def get_config_files(self):
         return ["config.yaml", "databricks/" + self.__databricks_config]
 
+    def modify_raw_config(self, raw_config: dict) -> dict:
```

Comment:
`dict` is not a type in python < 3.10, you have to use `Dict[type, type]`

Reply:
this is autogenerated from parent